gtd-timer/Services/LogInService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Security.Claims;
+using System.Text;
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+using System.Threading.Tasks;
+using gtdtimer.Constant;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace gtdtimer.Services
+{
+    public class LogInService : ILogInService
+    {
+        private ApplicationUserManager<User> userManager;
+        public LogInService()
+        {
+            this.userManager = userManager;
```

Comment:
this compiles? looks weird

Reply:
You are right, now this code isn't compiled, because class ApplicationUserManager isn't defined, @Taras Kataryna implement this class if I don't mistake.
I thought that need find user directly using ApplicationUserManager, but if you say use UoW, it's ok, I will rework.

Reply:
And I forgot add in constructor parameter of type ApplicationUserManager